dataland-frontend/src/components/forms/parts/elements/basic/CheckboxesListFormElement.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <FormKit
+    type="checkbox"
+    :key="key"
+    :name="name"
+    v-model="checkboxValue"
+    :options="options"
+    :validation="validation"
+    :outer-class="{
+      'yes-no-radio': true,
+    }"
+    :inner-class="{
+      'formkit-inner': false,
+    }"
+    :input-class="{
+      'formkit-input': false,
+      'p-radiobutton': true,
+    }"
+    :id="fieldName + name"
+    :ignore="true"
+    :plugins="[disabledOnMoreThanOne]"
+    @input="emitUpdateCurrentValue($event)"
+  />
+  <FormKit
+    type="text"
+    :name="name"
+    v-model="currentValue"
+    :validation="validation"
+    :validation-label="validationLabel"
+    :ignore="shouldBeIgnored"
+    :key="key"
+    :outer-class="{ 'hidden-input': true, 'formkit-outer': false }"
+  />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { FormKit } from "@formkit/vue";
+import { disabledOnMoreThanOne } from "@/utils/FormKitPlugins";
+
+export default defineComponent({
+  name: "CheckboxesListFormElement",
+  components: { FormKit },
+  data() {
+    return {
+      key: 0,
+      shouldBeIgnored: false,
+      currentValue: null,
+      checkboxValue: [] as Array<string>,
+    };
+  },
+  methods: {
+    disabledOnMoreThanOne,
+    /**
+     * Emits an event when the currentValue has been changed
+     * @param checkboxValue current value
+     */
+    emitUpdateCurrentValue(checkboxValue: [string]) {
+      if (checkboxValue[0]) {
+        this.shouldBeIgnored = false;
+        this.key++;
```

Comment:
Instead of `this.key++` could we use a computed value or watchers to somehow refresh these components? Or perhaps emitting/listening to an event... or somehow toggle `v-if`?